### DISCUSSION TITLE
Cant enable enforce https

### DISCUSSION THREAD

**Discussion by ftashish** [community/community#135745] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I cant enable enforce https in my git html css page its a new domain and i have configured the dns as tou suggested its bren 2 days please help asap

**Comment by jessdebrito** [comment]
Hi, @ftashish. 
The problem is probably in your DNS configuration. I recommend removing all DNS settings and re-adding them on by one. There may be a conflict or a duplicate line.
Link:
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site
Is important to remember: 1 - Your index.html need to be in the project root.  2 - DNS settings can take a while to propagate. Every time I config. it took around 1hour, maybe less. But some services ask to wait a day.
